### PR TITLE
feat(ui): add blue screen of death

### PR DESCRIPTION
**Pull request by rudolfs**
Closes: https://github.com/radicle-dev/radicle-upstream/issues/828.

Design as sketched out [here](https://www.figma.com/file/owmgsbs6lnUt8R1bixstCA/%F0%9F%8C%B1%F0%9F%86%99-Radicle-Upstream?node-id=5952%3A204). The copy could still further be tweaked \cc @abbey-titcomb.

demo

tweet

### CODE REVIEW

File: ui/Screen/Bsod.svelte [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+</style>
+
+<Fullscreen
+  escapable={false}
+  style="background-color: var(--color-secondary);"
+  contentStyle="display: flex; width: 312px; align-items: center;">
+  <div class="container">
+    <Emoji emoji="🧻" size="huge" style="margin-bottom: 1.5rem;" />
+    <p>
+      We're not totally sure what's going on, but we can't load the app right
```
Comment (FintanH) [inline ui/Screen/Bsod.svelte:33]:
```suggestion
      We're not totally sure what's going on, but we can't load the app right now
```

**Review by FintanH — CHANGES_REQUESTED** [review]
:)

**Review by FintanH — DISMISSED** [review]
:toilet:

**Comment by juliendonck** [comment]
🔥

**Comment by abbey-titcomb** [comment]
@brandonhaslegs @rudolfs @juliendonck — do we want "tweet at us" to be the main support action? We'd probably want support stuff to go to Discourse then to GitHub right? 

For example, I was planning on creating a "Support" category on Discourse — could be nice to have issues there instead of filling up the Twitter feed.

**Comment by rudolfs** [comment]
> For example, I was planning on creating a "Support" category on Discourse — could be nice to have issues there instead of filling up the Twitter feed.

@abbey-titcomb yeah, I agree. @brandonhaslegs and I just wanted to have something to start with to get the conversation going.

Could you provide me the link to which this button should be linking to?

**Comment by abbey-titcomb** [comment]
> Could you provide me the link to which this button should be linking to?

Sure, here's a Support category -> https://radicle.community/c/support/13

We can build out the copy to send people to GitHub or to Matrix chat based on what Support they need.

**Comment by juliendonck** [comment]
Do we wanna use github at all?

**Comment by abbey-titcomb** [comment]
> Do we wanna use github at all?

Yeah definitely, but to keep Issue list clean I would say we go from Discourse -> GitHub. Basically our team can send people from Discourse to GitHub if an issue is escalated. 

I also think there will be a sync support channel in the Matrix...

**Comment by rudolfs** [comment]
@abbey-titcomb updated.

Light mode:
Screenshot 2020-10-22 at 17 50 15

Dark mode:
Screenshot 2020-10-22 at 17 49 55

**Comment by FintanH** [comment]
I am _not_ loving that black on blue :grimacing:

**Comment by rudolfs** [comment]
> I am not loving that black on blue 😬

I know, talked to @brandonhaslegs he suggested painting it white, and so I did.

**Review by abbey-titcomb — APPROVED** [review]

**Review by xla — APPROVED** [review]
🐐 🚁 🐮 🛅

**Pull request merged**